app/addon_cmds/commands.py

```diff
@@ -198,4 +198,6 @@ def uninstall(ctx):
         parallel=ctx.obj["parallel"],
         timeout=ctx.obj["timeout"],
         rosa=ctx.obj["rosa"],
+        brew_token=ctx.obj["brew_token"],
+        api_host=ctx.obj["api_host"],
```

Comment:
`run_action` needs to accept some value for `brew_token` and `api_host` otherwise it will fail; it doesn't add any flag requirements for uninstall commands since `brew_token` is not required and `api_host` has default value

Reply:
They are only needed for `install_addon` ; give them defaults in the function declaration.

Reply:
done